src/index.ts

```diff
@@ -577,7 +598,7 @@ export class Pool extends (EventEmitter as new () => PoolEmitter) {
         new Promise((_, reject) => {
           connectionTimeoutTimer = setTimeout((): void => {
             reject(new Error('Timed out trying to connect to postgres'));
-          }, this.options.connectionTimeoutMillis);
+          }, this.options.waitForAvailableConnectionTimeoutMillis);
```

Comment:
I was under the impression that `connectionTimeoutMillis` is for query execution time, while this timeout was for getting the connection, so that's why I changed it to `waitForAvailableConnectionTimeoutMillis`

Reply:
After looking into how pg pool uses the `connectionTimeoutMillis` option, I think it's being used in two different ways. The pg pool library uses this to dictate how long an open connection will remain open, while postgres-pool is using this to control how long to wait for a new connection to be created.

Reply:
I believe it's ok how it's written. `connectionTimeoutMillis` specifies the threshold for how to try connecting to the postgres server, while `waitForAvailableConnectionTimeoutMillis` is how long a query should wait until a pool connection becomes available.